concepts/05 Widgets/SelectBox/00 Getting Started with SelectBox/25 Group Data.md

```diff
@@ -0,0 +1,202 @@
+The **SelectBox** can display data grouped by category. To make this possible, use the [DataSource]() component. Its API allows you to [sort](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#sort), [filter](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#filter), [select](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#select), and [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) data. At its core, the **DataSource** has a [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) - an object that keeps data and provides an API to access and modify it. To configure the store, use the **DataSource**'s [store]() object. Specify its [type]() as *"array"*, pass the initial data array to the [data]() field, and set the [key] field. 
+
+To group data, specify the data field to group by in the **DataSource**'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) option and set the **SelectBox**'s [grouped](/Documentation/ApiReference/UI_Widgets/dxSelectBox/Configuration/#grouped) option to **true**.
+
+
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#selectBox").dxSelectBox({
+            const data = [{
+                ID: 1,
+                Name: 'Banana',
+                Category: 'Fruits',
+            }, {
+                ID: 2,
+                Name: 'Cucumber',
+                Category: 'Vegetables',
+            }, {
+                ID: 3,
+                Name: 'Apple',
+                Category: 'Fruits',
+            }, {
+                ID: 4,
+                Name: 'Tomato',
+                Category: 'Vegetables',
+            }, {
+                ID: 5,
+                Name: 'Apricot',
+                Category: 'Fruits',
+            }]
+
+            var dataSource = new DevExpress.data.DataSource({
+                store: data,
+                type: 'array',
+                key: "ID",
+                group: "Category"
+            });
+
+            $("#selectBox").dxSelectBox({
+                dataSource: dataSource,
+                displayExpr: "Name",
+                searchEnabled: "true",
+                onValueChanged: function(e) {
+                    if (e.previousValue !== null) {
+                        console.log(e.previousValue.Name);
+                    }
+                    console.log(e.value.Name);
+                },
```

Comment:
The only change is the `grouped` option. There's no need to show full SelectBox configuration.